reserver/reserver_obj.py

```diff
@@ -29,7 +29,19 @@ def __init__(self, api_token, test_pypi=False):
         self.password = api_token
         self.test_pypi = test_pypi
 
-        
+
+    def batch_upload_to_pypi(self, *names):
```

Comment:
I prefer the input to be listed. I think that's a more pythonic solution.
Imagine a simple use case from the user's point of view. They should pass their desired package names list by `*pacakge_name` to the `batch_upload_to_pypi` which can be a bit weird for novice users.

Reply:
with this syntax, the user can do both of these:
`.batch_upload_to_pypi(NAME1, NAME2, NAME3)` and also `.batch_upload_to_pypi([NAME1, NAME2, NAME3])`
so you can easily pass the array you want to it.
@sadrasabouri if it's not the case, walk through it more.

Reply:
That's not true I guess. If you use the array call version it tries to call the `upload` method with the list itself instead of elements. The alternative is `.batch_upload(*[NAME1, NAME2, NAME3])`.

Reply:
ok sure I will investigate this, I think we both do have same intension, sure, I will take care of.

Reply:
Yes, you are right, thanks
I will take care of it.
@sadrasabouri